flexget/plugins/clients/transmission.py

```diff
@@ -1,6 +1,6 @@
 import base64
-import os
 import re
+import os
```

Comment:
Any reason to switch the order of this? We normally do alphabetical order. (and we have a tool, isort, which does the sorting of imports automatically)